src/commonMain/kotlin/com/sunnychung/lib/multiplatform/kdatetime/KDateTimeFormat.kt

```diff
@@ -198,8 +210,10 @@ class KDateTimeFormat(val pattern: String) {
 
     protected fun parseAmPm(input: String): Int {
```

Comment:
The lists `AMPM_NAMES_CAPS` and `AMPM_NAMES` can be supplied as an argument to the function `parseAmPm`, so that an input only maps to either `AMPM_NAMES_CAPS` or `AMPM_NAMES` at a time.

Reply:
I think that these values have default values already.
Maybe most users use `AMPM_NAMES_CAPS` and `AMPM_NAMES` at a time, but Does it need to be passed at once (single function)? I'm curious about your answer!

Reply:
In most cases, lowercase of `AMPM_NAMES_CAPS` equals with `AMPM_NAMES`, but I just considered what if they have differences. (In case: Non-english)

Reply:
If they have differences, then we should not let a non-caps token `a` maps to a caps name, or a caps token `A` maps to a non-caps name. Otherwise it looks like a bug, or we should not have introduced caps and non-caps.